connector/dialect.tdd

```diff
@@ -553,164 +285,19 @@
-    </function>
-    <function group='aggregate' name='COUNTD' return-type='int'>
-      <formula>COUNT(DISTINCT %1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='str' />
-    </function>
-    <function group='aggregate' name='COUNTD' return-type='int'>
-      <formula>COUNT(DISTINCT %1)</formula>
-      <unagg-formula>(case when %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='datetime' />
-    </function>
-    <function group='aggregate' name='COVAR' return-type='real'>
-      <formula>COVAR_SAMP(%1, %2)</formula>
-      <unagg-formula>NULL</unagg-formula>
-      <argument type='real' />
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='COVARP' return-type='real'>
-      <formula>COVAR_POP(%1, %2)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN NULL WHEN %2 IS NULL THEN NULL ELSE 0.0 END)</unagg-formula>
-      <argument type='real' />
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='bool'>
-      <formula>BOOL_OR(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='bool' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='real'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='numeric;logical' name='ZN' return-type='real'>
+      <formula>COALESCE(%1, 0)</formula>
       <argument type='real' />
     </function>
-    <function group='aggregate' name='MAX' return-type='int'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='numeric;logical' name='ZN' return-type='int'>
+      <formula>COALESCE(%1, 0)</formula>
       <argument type='int' />
     </function>
-    <function group='aggregate' name='MAX' return-type='str'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='str' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='datetime'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='datetime' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='date'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='date' />
-    </function>
-    <function group='aggregate' name='MIN' return-type='bool'>
-      <formula>BOOL_AND(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='bool' />
-    </function>
-    <function group='aggregate' name='MIN' return-type='real'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='MIN' return-type='int'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='numeric' name='DIV' return-type='int'>
```

Comment:
A question about division and arithmetic operators in general:
- MongoSQL generally supports `<any numeric> <arithmetic operator> <any numeric>`, but this file only specifies arithmetic operations were the operands have the same type (int, int or real, real). Do we need to specify the mixed versions (int, real; real, int)?

Reply:
For the `create calculated field` feature I am able to use the operators with int and real values without specifying the argument types.  I'm not sure where else Tableau will use the functions so I added the different variations of the operands.